internal/pkg/agent/cmd/install.go

```diff
@@ -125,6 +134,9 @@ func installCmd(streams *cli.IOStreams, cmd *cobra.Command, args []string) error
 
 	if enroll && fleetServer == "" {
 		if url == "" {
+			if nonInteractive {
+				return fmt.Errorf("missing required --url argument")
```

Comment:
[Suggestion]
Reword the message to focus on the fact that the "URL used to enroll the agent is missing" instead of just pointing out the cli flag. It's likely the user won't know what `--url` is for. Anyway to also cite the flag in the error message is good.

Reply:
```suggestion
  return fmt.Errorf("missing required --enrollment-token argument used to enroll the agent")
```
I've pushed the commits with the following changes